core/platform/cache/redis_cache_services.py

```diff
@@ -51,36 +53,46 @@ def get_memory_cache_stats():
         memory in bytes, peak memory usage in bytes, and the total number of
         keys stored as values.
     """
-    redis_full_profile = OPPIA_REDIS_CLIENT.memory_stats()
-    memory_stats = caching_domain.MemoryCacheStats(
+    # We have ignored [attr-defined] below because there is some error in
+    # the redis typeshed. Typestubs don't define the method memory_stats()
+    # for the redis.StrictRedis object.
```

Comment:
Btw, did we report this on the mypy repo, or do they already know about the issue?

Reply:
We have reported it. See [here](https://github.com/python/typeshed/issues/5906).